includes/discovery/sensors/runtime/apc.inc.php

```diff
@@ -15,3 +15,65 @@
     $high_limit         = 3000;
     discover_sensor($valid['sensor'], 'runtime', $device, $oid, $index, $type, $descr, $divisor, '1', $low_limit, $low_limit_warn, $warn_limit, $high_limit, $current);
 }
+
+// InRow IRRP100
+$oids = snmp_get($device, 'airIRRP100GroupSetpointsCoolMetric.0', '-OsqnU', 'PowerNet-MIB');
+if ($oids) {
+    echo 'APC InRow IRRP100 ';
+    // airIRRP100UnitRunHoursAirFilter
+    $index = 0;
+    $cur_oid = '.1.3.6.1.4.1.318.1.1.13.3.3.1.2.3.1.';
+    $current = snmp_get($device, 'PowerNet-MIB::airIRRP100UnitRunHoursAirFilter.'.$index, '-Oqv');
+    $service_interval = snmp_get($device, 'PowerNet-MIB::airIRRP100UnitServiceIntervalAirFilter.'.$index, '-Oqv');
+    $alarm_status= snmp_get($device, 'PowerNet-MIB::airIRRP100UnitServiceIntervalAirFilterAlarm.'.$index, '-Oqv');
+    $multiplier = 60;
+    $current = ($current * 60);
```

Comment:
You can just do `$current * $multiplier` in all of these so that changing `$multiplier` will be all that's needed (I know it won't change).